src/main/java/com/be3c/sysmetic/global/util/file/controller/FileController.java

```diff
@@ -0,0 +1,63 @@
+package com.be3c.sysmetic.global.util.file.controller;
+
+import com.be3c.sysmetic.global.util.file.dto.FileReferenceType;
+import com.be3c.sysmetic.global.util.file.dto.FileRequestDto;
+import com.be3c.sysmetic.global.util.file.dto.FileResponseDto;
+import com.be3c.sysmetic.global.util.file.service.FileServiceImpl;
+import org.apache.tomcat.util.http.fileupload.FileUploadException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.util.Map;
+
+/**
+ * 테스트용
+ */
+
+@RestController
+@RequestMapping("/api/file")
+public class FileController {
+
+    @Autowired
+    FileServiceImpl fileService;
+
+    @PostMapping("/upload")
+    public ResponseEntity<?> uploadFile(@RequestParam("file") MultipartFile file) throws FileUploadException {
```

Comment:
```suggestion
    public ResponseEntity<APIResponse<?>> uploadFile(@RequestParam("file") MultipartFile file) throws FileUploadException {
```
APIResponse 사용으로 바뀌어야 할 거 같습니다

Reply:
해당 클래스 상기 명시된 것처럼 테스트용입니다!